src/rgw/rgw_s3select.cc

```diff
@@ -610,6 +622,7 @@ int RGWSelectObj_ObjStore_S3::handle_aws_cli_parameters(std::string& sql_query)
   size_t _qo = m_s3select_query.find("<" + output_tag + ">", 0);
   size_t _qs = m_s3select_query.find("</" + output_tag + ">", _qi);
   m_s3select_output = m_s3select_query.substr(_qo + output_tag.size() + 2, _qs - (_qo + output_tag.size() + 2));
+  extract_format_by_tag(m_s3select_output, "JSON", output_json_format);
```

Comment:
please verify if it ("JSON") could be mixed with input-serialization.
i.e
upon setting input=JSON it will set also output=JSON

Reply:
In this case, If the input serialization is CSV or JSON and the output serialization is JSON, the output_json_format would be JSON.
Should we need to add the following condition?:
1: If the input serial is CSV or JSON and the output serial is CSV, CSV should be extracted.
or For every input serial is JSON, only the JSON output format is possible?

Reply:
the input/output-serialization indications do not depend on each other.
for JSON input the CSV output is possible (this is the current status).